### DISCUSSION TITLE
PAML Renaming

### DISCUSSION THREAD

**Discussion by danbryce** [Bioprotocols/labop#161] [Polls]
Please place your vote for one of the following names:

**Comment by schulz-m** [comment]
Protocol Modeling Language (PML) would be my preferred option :-D

**Reply by danbryce** [reply]
I'll accept write-in votes, if people want to upvote your suggestion. Thanks!

**Reply by vishalguptabt1** [reply]
I prefer PML too.

**Reply by photocyte** [reply]
Added PML to the table: https://github.com/Bioprotocols/paml/issues/107#issuecomment-1238236851

**Reply by jakebeal** [reply]
I'd like to withdraw my vote for POML and vote for PML.

**Comment by photocyte** [comment]
Also highlighting that many options & discussions on this were held in this related github issue: https://github.com/Bioprotocols/paml/issues/107
